.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "main"
+    ]
+}
```

Comment:
Usually these files in .vscode are pretty specific to your local dev environment, so they don't get checked in. Usually it's one of the first paths I add to my .gitignore.

That being said, this is a pretty minor issue.